backend/tests/projects/toolmodels/test_diagrams.py

```diff
@@ -65,13 +71,84 @@ def fixture_mock_git_diagram_cache_index_api(git_type: git_models.GitType):
             )
         case git_models.GitType.GITHUB:
             responses.get(
-                f"https://example.com/api/v4/repos/test/project/actions/artifacts/12347/zip",
+                "https://example.com/api/v4/repos/test/project/actions/artifacts/12347/zip",
                 status=200,
                 body=get_zipfile(),
                 content_type="application/zip",
             )
 
 
+@pytest.fixture(name="mock_git_diagram_cache_from_repo_api")
+def fixture_mock_git_diagram_cache_from_repo_api(
+    git_type: git_models.GitType, git_response_status: int
+):
+    match git_type:
+        case git_models.GitType.GITLAB:
+            responses.get(
+                "https://example.com/api/v4/projects/10000/repository/files/diagram_cache%2Findex.json?ref=diagram-cache%2Fmain",
+                status=git_response_status,
+                json={
+                    "file_name": "index.json",
+                    "file_path": "diagram_cache/index.json",
+                    "size": 2428,
+                    "encoding": "base64",
+                    "content_sha256": "ec00b56c2c87897dbd6e8d42de61b8ec6e3337cc442a09add86560920ec2872a",
+                    "ref": "main",
+                    "blob_id": "ee4b99b97f008dff549de9a40ddab139ea44435b",
+                    "commit_id": "1273b25fe671e014afb38d594825bb1a9d3206df",
+                    "last_commit_id": "85306aed4cf34c315ca3c31659fde02df9896e98",
+                    "execute_filemode": False,
```

Comment:
These are attributes are the same for all calls. Please store them in a variable instead.

Reply:
does it in general make sense to keep that here? It should mainly show the structure of a response while it is already stripped a bit - should we remove all currently unused attributes?